### PR TITLE
update docs with more simpler example

### PR DESCRIPTION
**Pull request by s3rj1k**
This example is more simple to understand

Signed-off-by: s3rj1k <evasive.gyron@gmail.com>

### CODE REVIEW

**Comment by osamingo** [comment]
@s3rj1k Plase resolve conflicts!

**Comment by s3rj1k** [comment]
@osamingo done, please do not disappear for long, planning to do more PRs in few days to come

**Comment by osamingo** [comment]
@s3rj1k OK! I'll review it if I free.

File: README.md [hunk 1]
```diff
@@ -60,6 +66,18 @@ func (h EchoHandler) ServeJSONRPC(c context.Context, params *fastjson.RawMessage
 	}, nil
 }
 
+func (h PositionalHandler) ServeJSONRPC(c context.Context, params *fastjson.RawMessage) (interface{}, *Error) {
+
+	var p PositionalParams
+	if err := Unmarshal(params, &p); err != nil {
```
Comment (osamingo) [inline README.md]: Please fix it! => `jsonrpc.Unmarshal` .

Comment (s3rj1k) [reply README.md]: fixed 04c6e90faac2675aa89e2176d2eec7d8

**Comment by osamingo** [comment]
@s3rj1k Good thanks!

**Pull request merged**